client/src/components/Product_page/product_info/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import './style.css'
+
+class ProductInfo extends Component {
+  state = {
+    product: {
+      img:
+        'https://media1.tenor.com/images/556e9ff845b7dd0c62dcdbbb00babb4b/tenor.gif?itemid=5300368'
+    },
+    name: 'loading'
+  }
+
+  request = (url, cb) => {
```

Comment:
use promise instead of cb ( as we agreed for this project )

Reply:
i fixed it in this commit  484ca19405125cdc5c30d2bb5fb121b92afaad0e